tasks/deploy/config/mainnet/morpho-blue-susde-dai-182day.ts

```diff
@@ -0,0 +1,126 @@
+
+const morphoBlueParameters = encodeAbiParameters(
+    [
+        {
+            components: [
+                {
+                    name: "morpho",
+                    type: "address",
+                },
+                {
+                    name: "collateralToken",
+                    type: "address",
+                },
+                {
+                    name: "oracle",
+                    type: "address",
+                },
+                {
+                    name: "irm",
+                    type: "address",
+                },
+                {
+                    name: "lltv",
+                    type: "uint256",
+                },
+            ],
+            name: "MorphoBlueParams",
+            type: "tuple",
+        },
+    ],
+    [
+        {
+            morpho: "0xBBBBBbbBBb9cC5e90e3b3Af64bdAF62C37EEFFCb" as `0x${string}`,
+            collateralToken:
+                "0x9D39A5DE30e57443BfF2A8307A4256c8797A3497" as `0x${string}`, // Mainnet sUSDe
+            oracle: "0x5D916980D5Ae1737a8330Bf24dF812b2911Aae25" as `0x${string}`,
+            irm: "0x870aC11D48B15DB9a138Cf899d20F13F79Ba00BC" as `0x${string}`,
+            lltv: BigInt("860000000000000000"),
+        },
+    ],
+);
+
+export const MAINNET_MORPHO_BLUE_DAI_14DAY: HyperdriveInstanceConfig<"MorphoBlue"> =
+    {
+        name: MAINNET_MORPHO_BLUE_DAI_14DAY_NAME,
+        prefix: "MorphoBlue",
+        coordinatorAddress: async (hre) =>
+            hre.hyperdriveDeploy.deployments.byName(
+                MAINNET_MORPHO_BLUE_COORDINATOR_NAME,
+            ).address,
+        deploymentId: toBytes32(
+            MAINNET_MORPHO_BLUE_DAI_14DAY_NAME.slice(0, 32),
+        ),
+        salt: toBytes32("0x420"),
+        extraData: morphoBlueParameters,
+        contribution: CONTRIBUTION,
+        fixedAPR: parseEther("0.083"),
+        timestretchAPR: parseEther("0.05"),
+        options: async (hre) => ({
+            extraData: "0x",
+            asBase: true,
+            destination: (await hre.getNamedAccounts())["deployer"] as Address,
+        }),
+        // Prepare to deploy the contract by setting approvals.
+        prepare: async (hre, options) => {
+            let pc = await hre.viem.getPublicClient();
+            let baseToken = await hre.viem.getContractAt(
+                "contracts/src/interfaces/IERC20.sol:IERC20",
+                MAINNET_DAI_ADDRESS,
+            );
+            let tx = await baseToken.write.approve([
+                hre.hyperdriveDeploy.deployments.byName(
+                    MAINNET_MORPHO_BLUE_COORDINATOR_NAME,
+                ).address,
+                CONTRIBUTION,
+            ]);
+            await pc.waitForTransactionReceipt({ hash: tx });
+        },
+        poolDeployConfig: async (hre) => {
+            return {
+                baseToken: MAINNET_DAI_ADDRESS,
+                vaultSharesToken: zeroAddress,
+                circuitBreakerDelta: parseEther("0.05"),
```

Comment:
TODO: Double-check this. If we raise the timestretch, we should  also increase this.

Reply:
We increased this to 10% so that we can support rates up to ~40% with uniform liquidity and rates over 100% on the very high end.